src/components/Products.tsx

```diff
@@ -133,13 +133,13 @@ function Products() {
                 switch (status) {
                     case "end of life":
                         return (
-                            <EuiBadge color="#FF4136" isDisabled={false}>
+                            <EuiBadge color="error" isDisabled={false}>
```

Comment:
If we'd want to reduce the code a bit, we could remove this switch (and the one in ProductBlocks) and define the components once  `<EuiBadge color={STATUS_TO_COLOR[status]} isDisabled={false}>` deriving the color from a hashmap/object like

```
const STATUS_TO_COLOR = {
    "end of life": "error",
    "active": "success",
    // etc
  }
``` 

I don't know if this is common practice in frontend though :)

Reply:
I changed the hard coded color: as it broke the dark theme. 

Most of `Products.tsx` and the other Metadata pages were implemented by interim's: refactored it ->

```
    const getStatusBadgeColor = (status: string) => {
        const statusColors: any = {
            "end of life": "error",
            "active": "success",
            "phase out": "danger",
            "pre production": "warning",
        }
        return statusColors.hasOwnProperty(status) ? statusColors[status] : "primary"
    }
```  

The default and the fact that spaces are used makes it a bit weird.

I try to keep all the specific color stuff in one place so the refactor to Emotion will be easier.